src/AppBundle/Resources/config/services.yml

```diff
@@ -1 +1,11 @@
-services: ~
\ No newline at end of file
+services:
+    sylius_demo.listener.admin_delete:
+        class: AppBundle\EventListener\AdminDeleteListener
+        arguments: ['@session', '@translator']
+        tags:
+            - { name: kernel.event_listener, event: sylius.admin_user.pre_delete, method: preDelete }
+    sylius_demo.listener.admin_update:
+        class: AppBundle\EventListener\AdminUpdateListener
```

Comment:
Actually we can squash this two listeners into a single listener with 2 methods.